<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_beach</class>
 <widget class="QDialog" name="add_beach">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>관광지 등록</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="local_input">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_input">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>122</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="toilet_input">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="shower_input">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="water_input">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="parking_input">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="local">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>local</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>name</string>
   </property>
  </widget>
  <widget class="QLabel" name="toilet">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>toilet</string>
   </property>
  </widget>
  <widget class="QLabel" name="shower">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>shower</string>
   </property>
  </widget>
  <widget class="QLabel" name="water">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>water</string>
   </property>
  </widget>
  <widget class="QLabel" name="parking">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>parking</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resist_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(0, 177,244);
	color: rgb(255, 255, 255);
    border-radius: 10px;
}
QPushButton:hover{
	
	background-color: rgb(255, 255, 255);
	color: rgb(0, 177, 244);
    border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>등 록</string>
   </property>
  </widget>
  <widget class="QPushButton" name="out_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(239, 41,41);
	color: rgb(255, 255, 255);
    border-radius: 10px;
}
QPushButton:hover{
	
	background-color: rgb(255, 255, 255);
	color: rgb(239, 41, 41);
    border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>나가기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>445</width>
     <height>350</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../다운로드/pngwing.com.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>local_input</zorder>
  <zorder>name_input</zorder>
  <zorder>toilet_input</zorder>
  <zorder>shower_input</zorder>
  <zorder>water_input</zorder>
  <zorder>parking_input</zorder>
  <zorder>local</zorder>
  <zorder>name</zorder>
  <zorder>toilet</zorder>
  <zorder>shower</zorder>
  <zorder>water</zorder>
  <zorder>parking</zorder>
  <zorder>resist_btn</zorder>
  <zorder>out_btn</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
